components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -107,32 +118,27 @@ export default function Preferences() {
                         </Alert>
                     )}
 
-                    {allowConfigureWorkspaceTimeout && (
-                        <>
+                    {maySetTimeout.data === true && (
+                        <form onSubmit={saveWorkspaceTimeout}>
```

Comment:
**praise**: Thanks for adding the form here! Looking forward to further improve this section here. Cross-posting relevant discussions[[1](https://gitpod.slack.com/archives/C01KPEPQYE6/p1676411374009249)][[2](https://gitpod.slack.com/archives/C01KPEPQYE6/p1676416083599729?thread_ts=1676411374.009249&cid=C01KPEPQYE6)][[3](https://gitpod.slack.com/archives/C01KPEPQYE6/p1676450040320919?thread_ts=1676411374.009249&cid=C01KPEPQYE6)] (internal) about the current interaction and future improvements. Cc @phimae @mikenikles